src/Resources/views/Admin/Core/user_block.html.twig

```diff
@@ -27,16 +27,18 @@ file that was distributed with this source code.
         </li>
 
         <li class="user-footer">
+            {% if sonata_user.userAdmin.hasAccess('edit', app.user) or sonata_user.userAdmin.hasAccess('show', app.user) %}
             <div class="pull-left">
                 <a href="{{
-                    sonata_user.userAdmin.isGranted('EDIT', app.user) ?
+                    sonata_user.userAdmin.hasAccess('edit', app.user) ?
```

Comment:
The isGranted I didnt like it a lot, looks like the normal is_granted from symfony but does a different thing. hasAccess looks cleaned, also uses same name 'edit' used in generate route. wdyt @VincentLanglet ?

Reply:
Seems ok